dockers/docker-snmp-sv2/start.sh

```diff
@@ -9,6 +9,15 @@ sonic-cfggen -d -y /etc/sonic/snmp.yml -t /usr/share/sonic/templates/snmpd.conf.
 mkdir -p /var/sonic
 echo "# Config files managed by sonic-config-engine" > /var/sonic/config_status
 
+CURRENT_HOSTNAME=`hostname`
+HOSTNAME=`sonic-cfggen -d -v DEVICE_METADATA[\'localhost\'][\'hostname\']`
```

Comment:
>HOSTNAME [](start = 0, length = 8)

Will you handle command error or empty hostname cases?

Reply:
> > HOSTNAME [](start = 0, length = 8)
> 
> Will you handle command error or empty hostname cases?

Added validation. Please check.